index.html

```diff
@@ -2,261 +2,341 @@
+
+        <li>
+          <ul class="description">
+            <div class="tonic">Multi-Post Stories</div>
+
+            <ul class="feature">
+              <li class="canopy">CANOPY</li>
+              <img src="./Graphics/Counter.svg" alt=".">
+              <li class="backend">Back End Dev</li>
+              <img src="./Graphics/Counter.svg" alt=".">
+              <li class="year">2015</li>
             </ul>
-            <div class="line"></div>
 
-        </section>
+          </ul>
+
+          <div class="contents">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+
+          <div class="madewith">
+            <div>html</div>
+            <div>css</div>
+            <div>javascript</div>
+          </div>
+
+          <div class="seeproject">
+            <p>See Project</p>
+          </div>
+        </li>
+      </ul>
+    </section>
+
+    <div class="fordesktop">
+      <section class="about">
+        <h3>About <br>Myself</h3>
+
+        <p>
+          Hello I’m a software developer! I can help you
+          build a product , feature or website Look through
+          some of my work and experience! If you like what
+          you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+
+        <h3 class="connect">
+          LET’S CONNECT
+        </h3>
+
+        <ul class="social">
+          <li class="icon twitter">
+            <a href="https://twitter.com/calebchris000"><img src="./Graphics/twitter.svg" alt="twitter"></a>
+          </li>
+
+          <li class="icon linkedin">
+            <a href="https://www.linkedin.com/in/caleb-nwaizu-b815aa23b/"><img src="./Graphics/Linkedin icon.svg"
+                alt="linkedin"></a>
+          </li>
+
+          <li class="icon medium">
+            <a href="https://medium.com/@calebchris000"><img src="./Graphics/medium.svg" alt="medium"></a>
+          </li>
+          <li class="icon github">
+            <a href="https://github.com/calebchris000"><img src="./Graphics/github.svg" alt="github"></a>
+          </li>
+
+          <li class="icon peace">
+            <a href="https://www.android.com/"> <img src="./Graphics/Vector.svg" alt="peace"></a>
+          </li>
+        </ul>
+
+        <div class="seeproject">
+          <p>Get my resume</p>
+        </div>
+      </section>
+  <div class="space"></div>
+      <section class="language">
+
+        <div class="language-header">
+          <li>Languages</li>
+          <li>
+            <img class="toggle" src="./Graphics/Normal Button/Tertiary/Icons/toggle.svg" alt="" />
+          </li>
+        </div>
+
+        <ul class="languages">
+
+          <div class="fordesk">
+            <div class="js-container">
+              <img src="./Graphics/Normal Button/Tertiary/Icons/javascript.png" alt="" />
+              <li class="javascript">JavaScript</li>
+            </div>
```

Comment:
- [ ] Kindly make the languages part match the design in Figma, they're aligned as a column and they have a gray `backround-color` 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/213454854-7ddfd6f6-1820-4594-a7e7-a3ca1cccb7f2.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/213454917-3c73ed4b-c6d6-49bb-9e89-c81df23437ef.png)